src/index.ts

```diff
@@ -0,0 +1,100 @@
+import Connection from './connection/index';
+import graphql, { IWeaviateClientGraphQL } from './graphql/index';
+import schema, { IWeaviateClientSchema } from './schema/index';
+import data, { IWeaviateClientData } from './data/index';
+import classifications, {
+  IWeaviateClientClassifications,
+} from './classifications/index';
+import batch, { IWeaviateClientBatch } from './batch/index';
+import misc, { IWeaviateClientMisc } from './misc/index';
+import c11y, { IWeaviateClientC11y } from './c11y/index';
+import { DbVersionProvider, DbVersionSupport } from './utils/dbVersion';
+import backup, { IWeaviateClientBackup } from './backup/index';
+import backupConsts from './backup/consts';
+import batchConsts from './batch/consts';
+import filtersConsts, { Operator } from './filters/consts';
+import cluster, { IWeaviateClientCluster } from './cluster/index';
+import clusterConsts from './cluster/consts';
+import replicationConsts from './data/replication/consts';
+import {
+  AuthAccessTokenCredentials,
+  AuthClientCredentials,
+  AuthUserPasswordCredentials,
+} from './connection/auth';
+import MetaGetter from './misc/metaGetter';
+
+export interface IConnectionParams {
+  authClientSecret?:
+    | AuthClientCredentials
+    | AuthAccessTokenCredentials
+    | AuthUserPasswordCredentials;
+  host: string;
+  scheme: string;
+  headers?: HeadersInit;
+}
+
+export interface IWeaviateClient {
+  graphql: IWeaviateClientGraphQL;
+  schema: IWeaviateClientSchema;
+  data: IWeaviateClientData;
+  classifications: IWeaviateClientClassifications;
+  batch: IWeaviateClientBatch;
+  misc: IWeaviateClientMisc;
+  c11y: IWeaviateClientC11y;
+  backup: IWeaviateClientBackup;
+  cluster: IWeaviateClientCluster;
+}
+const app = {
+  client: function (params: IConnectionParams): IWeaviateClient {
+    // check if the URL is set
+    if (!params.host) throw new Error('Missing `host` parameter');
+
+    // check if the scheme is set
+    if (!params.scheme) throw new Error('Missing `scheme` parameter');
+
+    // check if headers are set
+    if (!params.headers) params.headers = {};
+
+    const conn = new Connection(params);
+    const dbVersionProvider = initDbVersionProvider(conn);
+    const dbVersionSupport = new DbVersionSupport(dbVersionProvider);
+    const test = [1, 2, 3];
```

Comment:
What is this for?

Reply:
I don't think this is from my changes? 🤔 It's not something I added manually

Reply:
Weird, its [not in the current HEAD](https://github.com/weaviate/typescript-client/blob/main/index.ts#L50-L54) either way if you don't mind removing it 🙂

Reply:
Ah, you meant just the last line - and now I recall it - was testing linting/formatting 😆